sensor/common/networkflow/metrics/metrics.go

```diff
@@ -82,4 +83,15 @@ var (
 		Name:      "processes_listening_on_port_removed",
 		Help:      "Total number of processes listening on ports",
 	})
+	activeFlowsTrackerSize = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "active_network_flow_tracker_size",
```

Comment:
How about including the word `total`?
```suggestion
		Name:      "active_network_flows_total",
```

Reply:
Done. I've also updated the description and a few related things to match the new name 🙂